src/bin/vip-import-sql.js

```diff
@@ -400,7 +400,7 @@ void command( {
 	)
 	.option(
 		'in-place',
-		'Perform a search and replace operation on a local SQL file, save the results to the file, then import the updated file.'
+		'Perform a search and replace operation on the local input file, and save the results to the file prior to import.'
```

Comment:
Same as https://github.com/Automattic/vip-cli/pull/2079#discussion_r1826916772

Reply:
How about this:

```
.option(
		'in-place',
		'Overwrite the local input file with the results of the search and replace operation prior to import.'
	)
	.option(
		'output',
		'Clone the local input file and overwrite the clone with the results of the search and replace operation. Accepts a local file path. Overridden by --in-place if both options are run in the same command, and ignored if the command excludes a --search-replace operation.'
	)
```

Reply:
That looks good! 👍